File: apirest_python/routers/Restaurante.py
from fastapi import FastAPI
from pydantic import BaseModel

router= FastAPI (tags=["Restaurante"])

class Restaurante(BaseModel):
    id_restaurante: str
    nombre: str
    direccion: str
    telefono: str
'''    mesas: Mesa
    reservas: Reserva
    menu: Menu
'''

File: websocket_ruby/app/models/restaurante.rb
# app/models/restaurante.rb
require_relative './mesa'
require_relative './reserva'
require_relative './menu'

class Restaurante
  attr_accessor :id_restaurante, :nombre, :direccion, :telefono, :mesas, :reservas, :menu

  def initialize(id_restaurante:, nombre:, direccion:, telefono:, mesas: [], reservas: [], menu: nil)
    @id_restaurante = id_restaurante
    @nombre = nombre
    @direccion = direccion
    @telefono = telefono
    @mesas = mesas
    @reservas = reservas
    @menu = menu
  end
end
